actions.go

```diff
@@ -75,9 +75,9 @@ type CustomActionsWebhookActionUpdateInput struct {
 type CustomActionsTriggerDefinitionCreateInput struct {
 	Name        string          `json:"name"`
 	Description *graphql.String `json:"description,omitempty"`
-	Owner       graphql.ID      `json:"ownerId"`
-	Action      *graphql.ID     `json:"actionId,omitempty"`
-	Filter      *graphql.ID     `json:"filterId,omitempty"`
+	Owner       ID              `json:"ownerId"`
+	Action      ID              `json:"actionId"`
```

Comment:
I think we still want `omitempty` for `Action` and `Filter` since the are optional on create